.changeset/healthy-queens-prove.md

```diff
@@ -0,0 +1,7 @@
+---
+'@open-wc/scoped-elements': major
+---
+
+Add compatibility with lit-html 2 and lit-element 3.
+
+Please, not that `getScopedTagName` class and instance methods have been removed because they are not needed anymore.
```

Comment:
should we consider keeping them, returning the tag name itself? just for making the migration easier?

Reply:
ok brought it back and deprecated it